pkg/agent/interfacestore/interface_cache.go

```diff
@@ -64,7 +64,8 @@ const (
 // Todo: add periodic task to sync local cache with container veth pair
 
 type interfaceCache struct {
-	cache cache.Indexer
+	cache                        cache.Indexer
+	podInterfaceDeletionHandlers []chan *InterfaceConfig
```

Comment:
Actually, I am thinking to notify the Pod interface deletion event from Pod's CmdDel call. I don't know which is better, would you help give a suggestion? @tnqn